__tests__/app.test.js

```diff
@@ -60,3 +60,51 @@ describe("/api/topics", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("GET - status:200 - responds with article's data", () => {
+    test("should return an array of objects", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.result)).toBe(true);
+        });
+    });
+    test("should respond with an object with the right properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          body.result.forEach((article) => {
```

Comment:
Good start, we will also want to check the correct article id is returned for the endpoint